### PR TITLE
2.0.5 Branding

### PR DESCRIPTION
**Pull request by nagilson**
We will release 2.0.4 soon. We already know what will be in 2.0.5 so it was added to the changelog.

### CODE REVIEW

**Review by MiYanni — APPROVED** [review]

**Comment by nagilson** [comment]
Thank you! :)

**Comment by nagilson** [comment]
wait for the path setting pr to merge before merging.

**Pull request merged**
